content/articles/two-factor-authentication-enforcement.markdown

```diff
@@ -0,0 +1,78 @@
+---
+title: Enforce Two-Factor Authentication for a shared Account
```

Comment:
I suggest

> Enforce Two-Factor Authentication for all members of an account

The term _shared_ is ambiguous as we use it for sharing a domain. It's not really a shared account, it's just an account with multiple members.